### PR TITLE
Port to .Net standard & NuGet package

### PR DESCRIPTION
**Pull request by Atrejoe**
Hi Swifty,

I just did ported the lib to .Net standard, making it wider available (also .Net core).
This also makes it simpler to build a NuGet package.

Would you mind publishing this a package, of shall I?

### CODE REVIEW

File: OpenWeatherAPI.Tests/APITests.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	/// <summary>
+	/// Tests <see cref="API"/>
+	/// </summary>
+	public class APITests
+	{
+		[Fact()]
+		public void QueryTest_Success()
+		{
+			//Arrange
+			var api = new API("a94a22bf61976a8575caacbb14401dfa");
```
Comment (swiftyspiffy) [inline OpenWeatherAPI.Tests/APITests.cs]: Is this your API key, or is this dummy data?

File: README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ Console.WriteLine(string.Format("The temperature in {0}, {1} is currently {2} °
 This repository also has a sample project. Find it here: https://github.com/swiftyspiffy/OpenWeatherMap-API-CSharp/tree/master/OpenWeatherAPI%20Example
 
 ### Credits and Libraries Utilized
-- Newtonsoft.Json - JSON parsing class.  Used to parse Twitch API calls.
```
Comment (swiftyspiffy) [inline README.md:71]: ooooops

**Review by swiftyspiffy — CHANGES_REQUESTED** [review]
Thanks a bunch for your help on this, it's great. Ya I can publish it. Just wanted to confirm with you on the comment in here. 

Sorry for the delay on this :(

File: OpenWeatherAPI.Tests/APITests.cs (hunk 1 above)
Comment (Atrejoe) [reply OpenWeatherAPI.Tests/APITests.cs]: Oops, that is mine, I'have invalidated it at openweather, and I'll remove it.

Comment (Atrejoe) [reply OpenWeatherAPI.Tests/APITests.cs]: ..done

**Comment by swiftyspiffy** [comment]
Thanks again!

**Pull request merged**